tests/phpunit/test-page-cache.php

```diff
@@ -211,4 +211,46 @@ public function test_pantheon_cache_default_max_age_filter() {
 		// Remove the filter.
 		remove_all_filters( 'pantheon_cache_default_max_age' );
 	}
+
+	/**
+	 * Helper function to get the max-age from the Cache-Control header of a REST API response.
+	 * 
+	 * We're using a REST response here because it is a publicly accessible method that adds the Cache Control value from get_cache_control_header_value() to the response it returns.
+	 * 
+	 * @return int The max-age value.
+	 */
+	private function get_max_age_from_rest_dispatch() {
+		// Set up a rest request.
+		$response = new WP_REST_Response();
+	
+		// We're using the rest_post_dispatch callback here because it's a publicly accessible method that adds Cache Control to the response it returns.
+		$api_response = $this->pantheon_cache->filter_rest_post_dispatch_send_cache_control( $response );
+		$headers = $api_response->get_headers();
+		$cache_control_header_values = explode( ', ', $headers['Cache-Control'] );
+		$max_age = (int) str_replace( 'max-age=', '', $cache_control_header_values[1] );
+
+		return $max_age;
+	}
+
+	/**
+	 * Test the Cache-Control header max-age.
+	 */
+	public function test_cache_control_headers() {
```

Comment:
This should be two tests for better clarity when one fails. A test that tests multiple things is just asking for spending more time investigating an unexpected failure

Reply:
Disagree that they should be distinct tests, we're testing a single thing and multiple tests against the same thing feels like overkill. However, agree that clarity could be an issue. Added custom messages to each assertion to it's clearer what failed where.